Sortable.js

```diff
@@ -447,6 +447,12 @@
 
 			if (activeGroup.pull == 'clone') {
 				cloneEl = dragEl.cloneNode(true);
+				// < for Meteor compatibility
```

Comment:
Hi, and you can not do without it?
If every adds alike "crutch", the library turned into garbage.
Secondly, this place can simply be rewritten. So to me need a serious reason why it is necessary and can not be done "from the outside".